libs/sync/sync_test.go

```diff
@@ -67,6 +67,9 @@ func setupFiles(t *testing.T) string {
 	err = createFile(sub2, "g.go")
 	require.NoError(t, err)
 
+	err = createFile(sub2, "h.txt")
```

Comment:
I'd recommend creating these files and directories using for loops, to make the code terser.

Reply:
@shreyas-goenka we can't really because we need predictable structure and the files are located in different subdirectories so using loops will make it even less readable

Reply:
What do you think about simplifying it like this? The file tree structure is a lot more evident in this case.

```
for p in {
  []string{dir, "a.go"},
  []string{dir, "b.go"},
  ...
  []string{dir, ".databricks", "e.go"}
  ...
  []string{dir, "test", "sub1", "f.go"}
} {
  testutil.Touch(t, p...)
}
```

Reply:
Yes, that's a good one, thanks!